docs/ThemePaletteDocs.md

```diff
@@ -0,0 +1,131 @@
+# Project Overview
+
+As part of an ongoing effort to consolidate the code base, and centralize the way we do styling, we
+are tackling this project to respond to several problems.
+
+- Many components use extensive in-line styling, leading to repeated styles and a frustrating
+  experience updating styling.
+- There is a lack of standardization in styling, with many components partially using CSS, and
+  partially in-line styles, and different ways of doing colors.
+- There has been a desire to implement a dark mode theme, which is not feasible with the way we
+  currently implement colors.
+- Colors were not centralized, with some in `src/_vars.scss` and others in `src/theme.ts` used in
+  different places.
+- The recent brand refresh necessitated updating certain colors and components, which gave us a good
+  opportunity to rethink/refactor the way we do styling throughout the project.
+
+#### For these reasons we decided to tackle a couple of related projects:
+
+- Create palettes of all the gordon colors in `src/theme.ts` for light and dark modes using the
+  `extendTheme` API (replacing the old `createTheme` API)
+- Replace the old `ThemeProvider` with the `Experimental_CssVarsProvider` in
+  `src/components/providers`
+- Go through all components and convert in-line style to CSS where possible to centralize and clean
+  up styling
+- Modify CSS to reverence the new theme CSS variables provided by the `CssVarsProvider`, replacing
+  the old SASS variables defined within `src/_vars.scss`
```

Comment:
```suggestion
  the old SASS variables defined within `src/_vars.scss`
```

I think it would beneficial to make a small dictionary just so everyone is on the same page for terms that would be beneficial for us to know where/how you want things changed. 

i.e
`CSS` - Cascading Style Sheets `.css` 
    - generated by `.scss` module
`SASS` - Syntactically Awesome Stylesheet `.scss`
    - module 
...

Also I noted that you did not actually mention anything about `MUI` overwrites with how we should implement colors when using the `sx` tag. I don't know if your current implementation renders `sx` completely useless (don't think it should but I haven't been looking far too closely) but it would be nice to know what you want developers to do with MUI components that need overwrites

Reply:
For the dictionary, I think most terms are things future developers will need to research themselves.  Particularly for CSS, SASS, and MUI.  Those are easily searchable online, and the official docs will tell them more than this doc ever can.
However, if you think there are some specific terms I use that aren't clear, please let me know, because I want to make sure that anything specific to us is clearly defined.  Might be helpful to get some input from @EjPlatzer on this as well.

Reply:
Also, I added a note about the sx tag, I think in most cases it should be centralized to CSS, but I'm not certain that's always the best plan.  For our cleanup project though, that is what we have been doing (I know recim uses the sx tag a lot more than the rest of the codebase though)